test/test_sensu.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+require File.dirname(__FILE__) + '/helper'
+
+class TestSensu < Test::Unit::TestCase
```

Comment:
We're using minitest for this project, so you'll need to change `Test::Unit::TestCase` to `Minitest::Test`.